devices/BlueNrg2/BlueNrg2.cs

```diff
@@ -0,0 +1,246 @@
+﻿using System;
+using System.Device.Gpio;
+using System.Device.Spi;
+using System.Threading;
+using BlueNrg2.Aci;
+using Microsoft.Extensions.Logging;
+using nanoFramework.Logging;
+
+namespace BlueNrg2
+{
+    public class BlueNrg2 : IDisposable
```

Comment:
I am going to wait with doing this as the final interface for this isn't set in stone yet (mostly because it might be good to integrate this with the nanoFramework.Bluetooth package)